{
  "exploitdb": [],
  "itw": [
    {
      "id": "CVE-2020-28975",
      "description": "svm_predict_values in svm.cpp in Libsvm v324, as used in scikit-learn 0.23.2 and other products, allows attackers to cause a denial of service (segmentation fault) via a crafted model SVM (introduced via pickle, json, or any other model permanence standard) with a large value in the _n_support array. NOTE: the scikit-learn vendor's position is that the behavior can only occur if the library's API is violated by an application that changes a private attribute.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2020-28975",
      "reportURL": "https://github.com/cjlin1/libsvm/blob/9a3a9708926dec87d382c43b203f2ca19c2d56a0/svm.cpp#L2501",
      "timeStamp": "2020-12-03T18:50:00Z"
    },
    {
      "id": "CVE-2020-28975",
      "description": "svm_predict_values in svm.cpp in Libsvm v324, as used in scikit-learn 0.23.2 and other products, allows attackers to cause a denial of service (segmentation fault) via a crafted model SVM (introduced via pickle, json, or any other model permanence standard) with a large value in the _n_support array. NOTE: the scikit-learn vendor's position is that the behavior can only occur if the library's API is violated by an application that changes a private attribute.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2020-28975",
      "reportURL": "https://github.com/scikit-learn/scikit-learn/issues/18891",
      "timeStamp": "2020-12-03T18:50:00Z"
    },
    {
      "id": "CVE-2020-28975",
      "description": "svm_predict_values in svm.cpp in Libsvm v324, as used in scikit-learn 0.23.2 and other products, allows attackers to cause a denial of service (segmentation fault) via a crafted model SVM (introduced via pickle, json, or any other model permanence standard) with a large value in the _n_support array. NOTE: the scikit-learn vendor's position is that the behavior can only occur if the library's API is violated by an application that changes a private attribute.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2020-28975",
      "reportURL": "https://packetstormsecurity.com/files/160281/SciKit-Learn-0.23.2-Denial-Of-Service.html",
      "timeStamp": "2020-12-03T18:50:00Z"
    }
  ],
  "trickest": [
    {
      "id": "CVE-2020-28975",
      "url": "https://github.com/Live-Hack-CVE/CVE-2020-28975",
      "added_at": "2022-02-11T19:41:50Z"
    },
    {
      "id": "CVE-2020-28975",
      "url": "https://github.com/scikit-learn/scikit-learn/issues/18891",
      "added_at": "2022-02-11T19:41:50Z"
    },
    {
      "id": "CVE-2020-28975",
      "url": "https://packetstormsecurity.com/files/160281/SciKit-Learn-0.23.2-Denial-Of-Service.html",
      "added_at": "2022-02-11T19:41:50Z"
    }
  ],
  "nomisec": [],
  "nuclei": [],
  "metasploit": [],
  "holloways": [],
  "openpoc": [
    {
      "cve": "CVE-2020-28975",
      "url": "https://github.com/Live-Hack-CVE/CVE-2020-28975",
      "added_at": "2022-02-11T19:41:50Z",
      "score": 0,
      "template_for": ""
    },
    {
      "cve": "CVE-2020-28975",
      "url": "https://github.com/cjlin1/libsvm/blob/9a3a9708926dec87d382c43b203f2ca19c2d56a0/svm.cpp#L2501",
      "added_at": "2020-12-03T18:50:00Z",
      "score": 0,
      "template_for": ""
    },
    {
      "cve": "CVE-2020-28975",
      "url": "https://github.com/scikit-learn/scikit-learn/issues/18891",
      "added_at": "2020-12-03T18:50:00Z",
      "score": 0.4,
      "template_for": ""
    },
    {
      "cve": "CVE-2020-28975",
      "url": "https://packetstormsecurity.com/files/160281/SciKit-Learn-0.23.2-Denial-Of-Service.html",
      "added_at": "2020-12-03T18:50:00Z",
      "score": 0.4,
      "template_for": ""
    }
  ]
}
